content/hi/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,50 @@
+---
+title: कंटेनर रनटाइम इंटरफेस
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+CRI एक प्लगइन इंटरफ़ेस है जो क्यूबलेट को क्लस्टर घटकों को पुन: संकलित करने की आवश्यकता के बिना, विभिन्न प्रकार के कंटेनर रनटाइम का उपयोग करने में सक्षम बनाता है।
+
+आपको काम करने की जरूरत है
+{{<glossary_tooltip text="container runtime" term_id="container-runtime">}} चालू
+आपके क्लस्टर में प्रत्येक नोड, ताकि
+{{< glosary_tooltip text="kubelet" term_id="kubelet" >}} लॉन्च हो सकता है
+{{< glosary_tooltip text="Pods" term_id="pod" >}} और उनके कंटेनर।
```

Comment:
Typo in the `glossary_tooltip` shortcode name (leading to [build failures](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65723d0c4a8f33000888a1c8)). Please consider localizing the "text" parameter passed to the shortcode to Hindi.

```suggestion
{{< glossary_tooltip text="क्यूबलेट" term_id="kubelet" >}} लॉन्च हो सकता है
{{< glossary_tooltip text="पॉड्स" term_id="pod" >}} और उनके कंटेनर।
```